### DISCUSSION TITLE
Adjusting class names

### DISCUSSION THREAD

**Discussion by RedSparr0w** [digitalfortress-tech/typeahead-standalone#50] [Ideas]
Any chance of adding the ability to change the default class names or extend them?

I'm currently using `bootswatch` for site themes which has quite a few different styles and could easily style the list if I was able to adjust `tt-list` to be `dropdown-menu` instead or even `tt-list dropdown-menu`.

Would this be something that would be considered to add to the project?

**Comment by niketpathak** [comment]
Hello @RedSparr0w , This does sound like something users would want to do. Thank you for the PR. As mentioned therein, I plan to do a bit of refinement before creating a release. Thanks!
